versions.mk

```
@@ -18,4 +18,7 @@ vVERSION := v$(VERSION:v%=%)
 
 CUDA_VERSION := 12.2.2
 
+GOLANG_VERSION ?= 1.20.9
```

```suggestion
CUDA_VERSION := 12.2.2
GOLANG_VERSION ?= 1.20.9
```

nit: in ther projects we don't have a blank line between these.